data/crac/crac-impl/src/main/java/com/farao_community/farao/data/crac_impl/json/deserializers/UsageRuleDeserializer.java

```diff
@@ -109,6 +111,8 @@ private static OnConstraint deserializeOnConstraintUsageRule(JsonParser jsonPars
         while (!jsonParser.nextToken().isStructEnd()) {
 
             switch (jsonParser.getCurrentName()) {
+                case TYPE:
+                    break;
```

Comment:
at least jsonParser.nextToken(), otherwise you will get a NullPointerException at the next  `switch (jsonParser.getCurrentName())`
but i can also be removed :rocket: